server/db/models/user.js

```diff
@@ -3,8 +3,8 @@ const Sequelize = require('sequelize')
 const db = require('../db')
 
 const User = db.define('user', {
-  firstName: { type: Sequelize.STRING, allowNull: false },
-  lastName: { type: Sequelize.STRING },
+  firstName: {type: Sequelize.STRING, allowNull: false},
```

Comment:
our formatters are fighting again. can you figure out how to resolve?
I want spaces between brackets!

Reply:
I've adjusted this, hopefully in both PRs, let me go check this one too though, to make sure it's up to date.